framework/include/timesteppers/TimeStepper.h

```diff
@@ -151,6 +153,9 @@ class TimeStepper : public MooseObject, public Restartable, public ScalarCouplea
   /// True if dt has been reset
   bool _has_reset_dt;
 
+  /// Cumulative amount of steps that have failed
+  int _failure_count;
```

Comment:
```suggestion
  unsigned int _failure_count;
```
it s always positive as it s a count

Reply:
see the notes under the failureproblem.C, same thing

Reply:
you ll have to work around it. A count cant be negative, the class attribute cant be of the wrong type.

Reply:
you can always convert for your particular case. We dont want anyone inheriting this to take liberties with it